---
layout: default
title: Bilin Nong's CV
---

<h1 style="text-align: center;">Bilin Nong's Digital Curriculum Vitae</h1>

My actual CV can be downloaded [**here.**]({{ site.url }}/assets/Resume_bnong.pdf){:target="_blank"}

## Education  
----
<p style="float: right;"><b>Sep 2021 - June 2025</b></p>
**University of Toronto (St. George)**  
Honours Bachelor of Science - Statistical Science & Bioinformatics - ASIP Co-op Program

**GPA:**  3.96/4.00;  **Average:** A+


## Experience
----
<p style = "float:right;"><b>Apr 2023 - Present</b></p>

**Undergraduate Researcher with [Dr. Jochen Weile](https://www.linkedin.com/in/jochen-weile-7a665767/?originalSubdomain=ca) and [Prof. Fritz Roth](https://thedonnellycentre.utoronto.ca/faculty/frederick-roth)**

*[Lunenfeld-Tanenbaum Research Institute @Sinai Health, Toronto](https://www.lunenfeld.ca)*


* Re-processed the raw data underlying existing variant effect maps (atlas of the functional effects of all possible genetic variants on proteins) with the latest versions of the [TileSeqMave](https://github.com/rothlab/tileseqMave/tree/c60d66c92665467d38ab203b7b18a075e719be87), a bioinformatics pipeline for the calculation of fitness scores from sequencing reads and provides a suite of quality control visualizations.
* Compiled benchmark sets of variants with known pathogenicity from online databases and literature for each map.
* Evaluated the predictions made by different versions of variant effect maps using the precision-recall curve and use them to infer evidence strength for clinical interpretation by calculating the log-likelihood ratio for pathogenicity.
* Gave recommendations for optimizing the implementation of TileseqMave pipelines based on analysis results.
* Delivered presentation on the methodologies and key finding of this project to the entire lab and wrote a report. (My report can be viewed [**here.**]({{ site.url }}/assets/BCB330Report.pdf){:target="_blank"})

&nbsp;

<p style = "float:right;"><b>May 2023 - Present</b></p>

**Research Trainee with [Prof. Kuan Liu](https://www.kuan-liu.com) and [Prof. Kevin Thorpe](https://www.dlsph.utoronto.ca/faculty-profile/thorpe-kevin-e/)** 

*[Dalla Lana School of Public Health, Toronto](https://www.dlsph.utoronto.ca)* 


* Conducted a series of simulations in **R** statistical software to simulate Randomized Controlled Trials (RCTs) with different outcomes (continuous, binary, repeatedly measured) under different missing mechanisms.
* Applied common missing data handling methods such as complete case analysis, imputations and inverse probability weighting to each simulated data set, and assessed their performance by employing a range of evaluation metrics.
* Provided recommendations on how to deal with missing data in RCTs based on evaluation results.
* Acquired in-depth knowledge and expertise in missing data methodologies, as well as the design and analysis of RCTs.
* Merged and wrangled simulated datasets to created data visualizations in **R** using **dplyr** and **ggplot2**.
* Gave a poster presentation on research showcase Day. (My poster can be viewed [**here.**]({{ site.url }}/assets/SUDS_POSTER_new.pdf){:target="_blank"})


## Projects 

----
**Analysis of Sensory Factors \& Coffee Ratings**
* Investigated the correlation between the overall ratings of a cup of coffee and sensory aspects of coffee plus the bean species by fitting a multiple linear regression (MLR) model using the data from Coffee Quality Institute.
* Performed data analysis using R includes cleaning the raw data set, fitting the MLR model, checking the model assumptions via diagnostic plots, applying Box-Cox transformation to mitigate violated assumptions, conducting ANOVA and individual t test to check for significant linear relationship, and qualifying model goodness by a series of likelihood criteria.
* Utilized the tidyverse and car packages in R for efficient data summary, data analysis and visualization, showcasing proficiency in programming using R statistical software.
* Critically evaluated the model's limitations due to certain violations and the presence of problematic observations, demonstrating a thorough understanding of model reliability and data integrity.

&nbsp;

**Learning Management System Design**
* Designed and implemented a schema for a learning management system database, specifically tailored to support the functionalities of MarkUs --- a web application for the submission and grading of student assignments.
* Developed and executed complex SQL queries to facilitate data retrieval and analysis, demonstrating a deep understanding of relational database and SQL intricacies.
* Embedded SQL queries into Python using psycopg2 library, showcasing the ability to integrate SQL with a high-level programming language for efficient data handling and manipulation.
* Conducted testing and validation of database functionalities, ensuring accuracy and reliability of the data, and thereby facilitating insightful analytics for educational management and improvement.

&nbsp;

**Analysis of Hypertension & Low-Income Data in Toronto**
* Implemented data cleaning and standardization processes, selected appropriate statistical methods for analyzing the dataset.
* Utilized the object-oriented programming paradigm to implement functions in Python and designed comprehensive test cases for each function to ensure validate program functionality.
* Analyzed and identified significant correlations between hypertension rates and low-income levels at the neighborhood level in Toronto, providing valuable insights into public health and socio-economic dynamics.
* Enhanced expertise in Python programming and data analysis libraries, demonstrating a strong foundation in data manipulation, statistical analysis, and result interpretation.



## Technical Skills
----
I am comfortable with the following languages/applications:
* Python (Numpy, Pandas, Scikit, Matplotlib, SpaCy)
* R (tidyverse, SimStudy, doParallel, survey)
* Git/GitHub
* HTML & CSS
* Unix Shell
* LaTeX
* Microsoft Excel and Access





## Relevant Courses & Grades
----

My transcript can be viewed [**here.**]({{ site.url }}/assets/transcript.pdf){:target="_blank"}

&nbsp;

Here is an exhaustive list of relevant courses I have taken:
* MAT135 (Calculus I)
* MAT136 (Calculus II)
* MAT235 (Multivariable Calculus)
* MAT223 (Linear Algebra I)
* BIO130 (Molecular and Cell Biology)
* BIO230 (From Genes to Organisms)
* BCH242 (Intro Biochemistry)
* BCB330 (Special Project in Bioinformatics)
* HMB265 (General & Human Genetics)
* CHM136 (Intro Organic Chem I)
* CHM247 (Intro Organic Chem II)
* CSC108 (Intro to Computer Programming)
* CSC148 (Intro to Computer Science)
* STA237 (Probability, Statistics and Data Analysis I)
* STA238 (Probability, Statistics and Data Analysis II)






## Honors and Awards
----
<p style = "float:right;"><b>2023</b></p>

**Summer Undergraduate Data Science (SUDS) Research Program Award**

* Awarded to fund a summer research project for undergraduate students in data science field.

&nbsp;

<p style = "float:right;"><b>2022, 2023</b></p>


**Deanâ€™s List Scholar**

* Awarded to students who had excellent GPA in the University

&nbsp;
